test/unit/modules/keyboard.coffee

```diff
@@ -80,5 +80,20 @@ describe('Keyboard', ->
       expect(dom($('.ql-bold').get(0)).hasClass('ql-active')).toBe(true)
       expect(dom($('.ql-size').get(0)).value()).toBe(size)
     )
+
+    it('removeHotkey by name', ->
+      fn = @quill.getModule('keyboard').removeHotkey('BOLD')
+      expect(typeof fn).toBe('function');
+    )
+
+    it('removeHotkey by number', ->
```

Comment:
I don't think its necessary to support passing in a keycode (so the implementation can be removed too).